files/acme/project/project.yml

```diff
@@ -0,0 +1,7 @@
+---
+kind: ProjectRequest
+apiVersion: v1
+metadata:
+  name: acme
```

Comment:
Same comment above - probably should use a template to allow for a custom name of the namespace

Reply:
With the resent merge (thank you), you can now just reference the generic projectrequest template:
https://raw.githubusercontent.com/redhat-cop/cluster-lifecycle/master/files/projectrequest/template.yml
... and supply a params file with these values